content/participants/ttjc@OptimusRahul/index.md

```diff
@@ -47,8 +47,49 @@ It would be good if someone helps and suggest me the best practices while workin
 
 ### projects
 
-I will start sharing my projects with live links and source code links after I finish implementing them.
+### Covid 19 Tracker
+
+This is react based web app, that give covid 19 status, of all countries.
+
+- You can find it [here](https://covidupdates.netlify.app/)
+- Github repo can be found [here](https://github.com/OptimusRahul/Covid19-Tracker)
+
+### Songify
+
+This is react based music player
+
+- You can find it [here](https://songify-player.netlify.app/)
+- Github repo can be found [here](https://github.com/OptimusRahul/Songify)
+
+### Open Source Project Finder
+
+This is javascript based website, that helps you to find all the open source project.
+
+- Github repo and preview can be found [here](https://github.com/OptimusRahul/Open-Source-Project-Finder)
```

Comment:
Where's the hosted link? It's a website, right?

Reply:
Yes, it's website, I was facing issue to deploy it over netlify, so I had attached the preview of the project. Now I have finally hosted it over heroku.
link - https://open-source-project-finder.herokuapp.com/
-[App](https://open-source-project-finder.herokuapp.com/)

Should I raise a pull request again?